docs/docusaurus/docs/cloud/set_up_gx_cloud.md

```diff
@@ -60,7 +60,7 @@ Environment variables securely store your GX Cloud access credentials. The GX Cl
 2. Run the following code to set the `GX_CLOUD_ACCESS_TOKEN` and `GX_CLOUD_ORGANIZATION_ID` environment variables, install GX Cloud and its dependencies, and start the GX Cloud agent:
 
     ```bash title="Terminal input"
-    docker run -e GX_CLOUD_ACCESS_TOKEN="<user_access_token>" -e GX_CLOUD_ORGANIZATION_ID="<organization_id>" greatexpectations/agent
+    docker run --rm -t -e GX_CLOUD_ACCESS_TOKEN="<user_access_token>" -e GX_CLOUD_ORGANIZATION_ID="<organization_id>" greatexpectations/agent
```

Comment:
I'm not convinced `-t` should be there by default

Reply:
Agreed. Removed it.